src/store/schools.unit.ts

```diff
@@ -77,27 +77,46 @@ describe("schools module", () => {
 				const setSchoolSpy = jest.spyOn(schoolsModule, "setSchool");
 				const setLoadingSpy = jest.spyOn(schoolsModule, "setLoading");
 
+				await schoolsModule.fetchSchool();
+
+				expect(setLoadingSpy).toHaveBeenCalledTimes(2);
+				expect(setLoadingSpy.mock.calls[0][0]).toBe(true);
+				expect(setSchoolSpy).toHaveBeenCalled();
+				expect(setSchoolSpy.mock.calls[0][0]).toStrictEqual(mockSchoolResponse);
+				expect(setLoadingSpy.mock.calls[1][0]).toBe(false);
+			});
+
+			it("should set school state correctly", async () => {
+				const mockMe = meResponseFactory.build({
+					school: { id: "sampleSchoolId" },
+				});
```

Comment:
```suggestion
				const mockMe = meResponseFactory.build();
```
Genauso beim Test dadrüber. Die Id spielt keine Rolle, muss also nicht explizit gesetzt werden.